vscode-extensions.json

```diff
@@ -160,6 +160,10 @@
     {
       "repository": "https://github.com/zowe/vscode-extension-for-zowe",
       "revision": "v1.5.1"
+    },
+    {
+      "repository": "https://bitbucket.org/atlassianlabs/atlascode",
+      "revision": "2.6.6"
```

Comment:
FYI the file needs to be sorted
```
 {
   "extensions": [
     {
+      "repository": "https://bitbucket.org/atlassianlabs/atlascode",
+      "revision": "2.6.6"
+    },
+    {
       "repository": "https://github.com/asciidoctor/asciidoctor-vscode",
       "revision": "v2.7.7"
     },
@@ -160,10 +164,6 @@
     {
       "repository": "https://github.com/zowe/vscode-extension-for-zowe",
       "revision": "v1.5.1"
-    },
-    {
-      "repository": "https://bitbucket.org/atlassianlabs/atlascode",
-      "revision": "2.6.6"
     }
   ]
 }
```

Reply:
Yes, I got an error when running `.build.sh`